### DISCUSSION TITLE
能否详细解释下 -c 参数的用法

### DISCUSSION THREAD

**Discussion by XuuChannel** [1715173329/UnblockNeteaseMusic#1031] [Q&A] [answered]
### 需求描述

没看懂（

**Comment by 1715173329** [comment, marked as answer]
这个是以前遗留下来的，用 bridge.js 单独运行一个查询音源信息的 api 接口，app.js 只做 MITM 和转发。
用例是在国内 vps 上面查询音源信息（地区限制），然后在海外 vps 上面运行主程序。
虽然我也不知道为什么会有人这么用
